DigitalOffice/Services/FileService/Mappers/FileMapper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using LT.DigitalOffice.FileService.Database.Entities;
+using LT.DigitalOffice.FileService.Mappers.Interfaces;
+using LT.DigitalOffice.FileService.Models;
+using LT.DigitalOffice.FileService.RestRequests;
+using System;
+
+namespace LT.DigitalOffice.FileService.Mappers
+{
+    public class FileMapper : IMapper<DbFile, File>, IMapper<FileCreateRequest, DbFile>
+    {
+        public File Map(DbFile dbFile)
+        {
+            if (dbFile == null)
+            {
+                throw new ArgumentNullException();
```

Comment:
Why not add the argument name to the exception message? Use namof(dbFile)

Reply:
Fixed